UITests/ReopenLastTabTests.swift

```diff
@@ -4,7 +4,6 @@
 
 import Foundation
 import Storage
-import EarlGrey
```

Comment:
This test suite has been disabled for long time, it would need some work to update the test with the current behaviour. Leaving the EarlGrey functions and so in the test in case it is re-enabled or moved to XCUI to have the basis of the tests